content/webapp/views/components/ScrollContainer/ScrollContainer.Navigation.tsx

```diff
@@ -118,7 +118,8 @@ const ScrollableNavigation: FunctionComponent<Props> = ({
         onClick={() => scrollByChildImageWidth('left')}
       >
         <Icon icon={arrowSmall} rotate={180} />
-        Prev
+        <span aria-hidden="true">Prev</span>
+        <span className="visually-hidden">Previous</span>
```

Comment:
```suggestion
        <span>Prev<span className="visually-hidden">ious</span></span>
```

(Pls know this is joke, this PR is great and I'm feeling very end of day 😄 )